packages/core/src/main/Account.ts

```diff
@@ -352,10 +357,22 @@ export class Account extends EventEmitter {
     const loadedClient = await this.service!.client.getLocalClient();
     await this.apiClient.api.client.getClient(loadedClient.id);
     this.apiClient.context!.clientId = loadedClient.id;
+    if (this.enableMLS) {
+      this.coreCryptoClient = await this.createMLSClient(loadedClient);
+    }
 
     return loadedClient;
   }
 
+  private async createMLSClient(client: RegisteredClient): Promise<CoreCrypto> {
+    const {CoreCrypto} = await import('@otak/core-crypto');
+    return CoreCrypto.init({
+      path: 'path/to/database',
+      key: 'a root identity key (i.e. enclaved encryption key for this device)',
```

Comment:
just curious about hese hardcoded values right now- are these just filler for the time being?

Reply:
Yes, they are placeholders. As of now, the core-crypto bundle doesn't have a database and doesn't need this part to work. We will fill them with the relevant data when it's time to do so :)